src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -845,25 +845,25 @@ WatchKey register(WatchService watcher,
      *
      * <p> An invocation of this method behaves in exactly the same way as the
      * invocation
-     * <pre>
+     * {@snippet lang=java :
      *     watchable.{@link #register(WatchService,WatchEvent.Kind[],WatchEvent.Modifier[]) register}(watcher, events, new WatchEvent.Modifier[0]);
```

Comment:
Take out the {\@link, it isn't needed and gets included literally.  (And make the line very long).

Reply:
Thanks. I fixed another place where a `@link` was included literally but missed this one. Fixed in [11ad946](https://github.com/openjdk/jdk/pull/13870/commits/11ad9465a3e7bc983c59eea96e7fd460b595a8bf).